Breeze.UI/main.ts

```diff
@@ -115,14 +117,19 @@ function startApi() {
       apipath = path.join(__dirname, '.\\assets\\daemon\\Breeze.Daemon.exe');
   }
 
+  //Handle spaces in paths
+  apipath = apipath.replace(/ /g, '\\ ');
+
```

Comment:
Not directly, but you weren't the only one to suggest it exactly like that: https://stackoverflow.com/questions/16395612/unable-to-execute-child-process-exec-when-path-has-spaces#26998326